build.gradle

```diff
@@ -84,6 +84,9 @@ allprojects {
         maven {
             url "https://repository.apache.org/snapshots"
         }
+        maven {
+            url 'https://jitpack.io'
```

Comment:
Why not read from Maven Central itself? The library seemed to be available there: https://mvnrepository.com/artifact/com.github.everit-org.json-schema/org.everit.json.schema/1.12.1?

Reply:
actually i based the repo on their README:

https://github.com/everit-org/json-schema#maven-installation

I am not sure if its the same on maven, but i see 62 other projects have dependency on maven artifact, so will use that.